app/views/shared/_students_table.erb

```diff
@@ -0,0 +1,12 @@
+<table class="table table-striped table-hover students-table">
+  <thead>
+    <tr>
+      <th>Last Name</th>
+      <th>First Name</th>
+      <th>Date of Birth</th>
+    </tr>
+  </thead>
+  <% students.each do |student| %>
```

Comment:
I believe you can just do `<%= students %>` here and it'll do a join of `render partial '_student', locals: {student: student}`.